src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements State {
+    private double radius = 0;
+    private String color = BLACK;
```

Comment:
Set these in the constructor
```suggestion
    private double radius;
    private String color;
```